client/v2/internal/buf.lock

```diff
@@ -4,16 +4,20 @@ deps:
   - remote: buf.build
     owner: cosmos
     repository: cosmos-proto
+    branch: main
     commit: 1935555c206d4afb9e94615dfd0fad31
   - remote: buf.build
     owner: cosmos
     repository: cosmos-sdk
-    commit: 44369accb517453db1f2228530da42c0
+    branch: main
+    commit: 86d2a697b026488089f13a71ceb3815c
   - remote: buf.build
     owner: cosmos
     repository: gogo-proto
-    commit: 6652e3443c3b4504bb3bf82e73a7e409
+    branch: main
+    commit: bee5511075b7499da6178d9e4aaa628b
   - remote: buf.build
     owner: googleapis
     repository: googleapis
-    commit: 62f35d8aed1149c291d606d958a7ce32
+    branch: main
+    commit: 40f07f5b563941f2b20b991a7aedd53d
```

Comment:
[note] the commit looks different from upstream: https://github.com/cosmos/cosmos-sdk/blob/2a48f6d7a24b728ac64eafe0929bb8893cc23ccf/client/v2/internal/buf.lock#L16-L19

although I'm not if it is necessary for these to match exactly